resources/rgbscripts/lines.js

```diff
@@ -33,7 +33,7 @@ var testAlgo;
     algo.linesAmount = 3;
     algo.properties.push("name:linesAmount|type:range|display:Amount|values:1,200|write:setAmount|read:getAmount");
     algo.linesSize = 10;
-    algo.properties.push("name:linesSize|type:range|display:Size|values:1,32|write:setSize|read:getSize");
+    algo.properties.push("name:linesSize|type:range|display:Size|values:1,32|write:setLinesSize|read:getLinesSize");
     algo.linesVariablity = 0;
```

Comment:
I think this is a typo and should be fixed everywhere in the script

Reply:
In the quest to find and fix the bugs in this script, I applied best practice to name the getters by the object they get and to avoid any name clashes for setSize / setSize in any other shadowed namespace.
The variable name in the script was not changed from "linesSize", so the getters and setters are [gs]etLinesSize by intention.
Would you like to revert this change in the getter and setter name or do you refer to the other best practice to use singular for the variable names, i.e. rename linesSize to lineSize?

Reply:
My comment is about "linesVariablity". Should be "linesVariability"

Reply:
Ah, thanks for the clarification. Blindness on my side. The script was broken due to this typo, but I updated it by consistently promoting the typo spelling instead of the correct one. I'll fix this.